crates/subspace-farmer/src/commitments.rs

```diff
@@ -293,18 +295,22 @@ impl Commitments {
             .collect::<Vec<Salt>>();
 
         for salt in salts {
-            let commitment_databases = self.inner.commitment_databases.lock().await;
-            let db_entry = match commitment_databases.databases.peek(&salt).cloned() {
+            let db_entry = match self
+                .inner
+                .commitment_databases
+                .lock()
```

Comment:
Previously, this lock was released after acquiring the lock `db_guard`. Now, this lock is released after the `match` statement completes?

Reply:
Yes